package.json

```diff
@@ -27,6 +27,7 @@
     "dotenv": "^8.2.0",
     "emoji-name-map": "^1.2.8",
     "github-username-regex": "^1.0.0",
+    "node-fetch": "^2.6.1",
```

Comment:
Why was node-fetch added? I can't seem to find where it was used and we already have axios for data fetching

Reply:
```suggestion
```

Reply:
I will remove it.